interfaces/mimir_worker/v0/README.md

```diff
@@ -0,0 +1,71 @@
+# `mimir_worker`
+
+## Usage
+
+This relation interface is used by [the Mimir Coordinator charm](/url) and [the Mimir Worker charm](/url) to exchange data.
```

Comment:
Not sure if this phrasing is clearer?
```suggestion
This relation interface is used by [the Mimir Coordinator charm](/url) and [the Mimir Worker charm](/url) to exchange Mimir Configuration data.
```

And I would remove the three bullet points below as they are re-write of the "Behavior" section.